docs/privacy-policy.rst

```diff
@@ -0,0 +1,361 @@
+.. This is linked from the footer of readthedocs.org
+.. and from the version (flyout) menu on docs sites
+
+Privacy Policy
+==============
+
+Effective date: **May 25, 2018**
+
+Welcome to Read the Docs.
+At Read the Docs, we believe in protecting the privacy of our
+users, authors, and readers.
+
+
+The short version
+-----------------
+
+We collect your information only with your consent;
+we only collect the minimum amount of personal information that is necessary
+to fulfill the purpose of your interaction with us;
+we don't sell it to third parties;
+and we only use it as this Privacy Policy describes.
+
+Of course, the short version doesn't tell you everything,
+so please read on for more details!
+
+
+Our services
+------------
+
+Read the Docs is made up of:
+
+readthedocs.org
+    This is a website aimed at documentation authors writing and building
+    software documentation. This Privacy Policy applies to this site
+    in full without reservation.
+
+readthedocs.io and other domains ("Documentation Sites")
+    These public websites are where Read the Docs hosts documentation on
+    behalf of documentation authors. A best effort is made to apply
+    this Privacy Policy to these sites but the documentation
+    may contain content and files created by documentation authors.
+
+readthedocs.com
+    This website is a commercial hosted offering for hosting private
+    documentation for corporate clients. It is governed by a separate
+    `policy and terms <https://readthedocs.com/terms/>`_.
```

Comment:
We have a separate terms of service, but I think we can use a common privacy policy.

Reply:
Ahhh I see...